run-system-tests/action.yml

```diff
@@ -203,14 +203,59 @@ runs:
         printf '#!/bin/bash\nset -o pipefail\nCONFIG_PATH=./zbox_config.yaml go test -timeout=60m -run  "^Test___Flaky.*$" ./... -json -count=1 | sed -r "/(=== (CONT|RUN|PAUSE).*)|(--- FAIL:.*)|(\\"Test\\":\\".*\/[pP]arallel\\")/d"' > FLAKY_TEST_RUNNER_COMMAND.sh && chmod 777 FLAKY_TEST_RUNNER_COMMAND.sh
+
+        mkdir -p ${BRANCH_DIR}/${GITHUB_SHA}/api
+        mkdir -p ${BRANCH_DIR}/${GITHUB_SHA}/cli
+        mkdir -p ${BRANCH_DIR}/${GITHUB_SHA}/flaky_cli
+        mkdir -p ${BRANCH_DIR}/latest/api
+        mkdir -p ${BRANCH_DIR}/latest/cli
+        mkdir -p ${BRANCH_DIR}/latest/flaky_cli
+
+        cd tests/cli_tests
         (./zwallet version --configDir ./config --config ./zbox_config.yaml --wallet ../ignore --silent | grep -A2 'Version info' | sed "s/Version info:/ZWallet Version Info:/") || true
         (./zbox version --configDir ./config --config ./zbox_config.yaml --wallet ../ignore --silent | grep -A2 'Version info' | sed "s/Version info:/ZBox Version Info:/") || true
 
+    - name: "Run API CLI System Tests"
+      shell: 'script --return --quiet --command "bash {0}"'
+      if:  ${{ env.RUN_SUBSET_OF_TESTS == 'false' }}
+      run: |
+        echo
+        echo "======================================================"
+        echo "RUNNING API SYSTEM TESTS:"
+        echo "======================================================"
+        echo
+
+        cd tests/api_tests
+        api_system_tests_exit_code=0
+        ~/go/bin/gotestsum --jsonfile test-output.json --hide-summary=output --format testname ${{ env.SHOULD_RETRY_FAILURES }} --raw-command -- ../../API_TEST_RUNNER_COMMAND.sh || api_system_tests_exit_code=$?
+        cat test-output.json | ~/go/bin/go-test-report --groupSize 1 --output ../../${{ env.BRANCH_DIR }}/${GITHUB_SHA}/api/index.html --title "0Chain API System test suite [${{ env.BRANCH_DIR }}/${GITHUB_SHA:0:8}] ran against [${{ env.NETWORK_URL }}] at [${{ env.TEST_TIME }}]"
+
+        cp -R ../../${{ env.BRANCH_DIR }}/${GITHUB_SHA}/api ../../${{ env.BRANCH_DIR }}/latest/api
+        if [[ $api_system_tests_exit_code == 0 ]];
+          then
+            echo API_TESTS_PASSED=true >> $GITHUB_ENV
+          else
+            echo API_TESTS_PASSED=false >> $GITHUB_ENV
+            echo "::error title=API System tests faled!::API System tests failed. Ensure tests are running against the correct images/branches and rule out any possible code issues before attempting a re-run"
+        fi
+
+        exit $api_system_tests_exit_code
+
+    - name: "Archive API Test Config & Console Output"
+      if: ${{ env.RUN_SUBSET_OF_TESTS == 'false' }}
+      uses: actions/upload-artifact@v2
+      with:
+        name: Flaky-System-Test-${{ env.TEST_TIME }}
```

Comment:
Should this be something like API-System-Test-${{  env.TEST_TIME }}?

Reply:
Good point I'll resolve now